tests/e2e/s3/utils.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python
+
+import subprocess
+import json
+import os
+import sys
+import boto3
+import time
+
+
+def update_agent_biling_clients(billing):
+    config = subprocess.check_output(
+        [
+            "kubectl",
+            "get",
+            "configmap",
+            "autoscaler-agent-config",
+            "-n",
+            "kube-system",
+            "-o",
+            "json",
+        ]
+    )
+    config = json.loads(config)
+    print(config)
+    agent_config_json = config["data"]["config.json"]
+    agent_config = json.loads(agent_config_json)
+    agent_config["billing"] = billing
+    agent_config_json = json.dumps(agent_config)
+    config["data"]["config.json"] = agent_config_json
+
+    result = subprocess.check_call(
+        [
+            "kubectl",
+            "patch",
+            "configmap",
+            "autoscaler-agent-config",
+            "-n",
+            "kube-system",
+            "--type",
+            "merge",
+            "-p",
+            json.dumps(config),
+        ]
+    )
+    assert result == 0
```

Comment:
`assert` only gets run in debug mode — should this instead be an explicit check & `raise Exception(...)` ?

(and I guess similar things for asserts lower down)